content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Scale Kubernetes Cluster using HPE GreenLake Terraform Provider
+date: 2022-07-12T14:06:02.781Z
+author: Arshad Saquib C L
+authorimage: /img/mypic192x192.jpg
+tags:
+  - " hpe-greenlake, devops, terraform, open source"
```

Comment:
For tags, you will need to create individual tag for each
Tag1: hpe-greenlake
tag 2: devops
tag 3: terraform
tag 4: open source